MergerLogic/DataTypes/Data.cs

```diff
@@ -104,6 +114,10 @@ protected Data(IServiceProvider container, DataType type, string path, int batch
             }
         }
 
+        protected abstract GridOrigin DefaultOrigin();
+
+        protected abstract Grid DefaultGrid();
```

Comment:
consider making it virtual instead as most types are 2X1

Reply:
Made virtual.